kaos/tuples.py

```diff
@@ -5,6 +5,7 @@
 
 from collections import namedtuple
 
+# pylint: disable=pointless-string-statement
 """Representation a 3D cartesian vector"""
```

Comment:
Why is this a string statement?

Reply:
That was meant to be a docstring. Upon further research, apparently this won't work for Sphinx. Pushing an update with fix.

ref: https://stackoverflow.com/a/28568351

Reply:
Tyty